scripts/test

```diff
@@ -1,3 +1,21 @@
 #!/usr/bin/env sh
 
-nvim --headless -u tests/init.lua -c "PlenaryBustedDirectory tests {minimal_init = 'tests//init.lua', sequential = true}"
+set -e
+
+err=0
+
+# Test without nix command
+export NIX_CONF_DIR="/dev/null"
+nvim --headless -u tests/init.lua -c "PlenaryBustedDirectory tests {minimal_init = 'tests//init.lua', sequential = true}" || err=1
```

Comment:
What is the purpose of testing without nix command? I don't think we have tests that check when nix is not available?

Also it looks the test suite now runs 3 times which seems a bit wasteful.